### PR TITLE
Attempt to fix formatting of examples

### PR DESCRIPTION
**Pull request by ImJasonH**
Before this change: 
Screen Shot 2019-12-17 at 2 40 51 PM

After this change:
https://github.com/ImJasonH/cli/blob/examples/docs/cmd/tkn_clustertask_delete.md

Also try to standardize on `'` instead of `"`, fix some typos, add some `:`s.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [x] Run the code checkers with `make check`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: pkg/cmd/pipeline/logs.go [hunk 1]
```diff
@@ -48,19 +48,22 @@ func logCommand(p cli.Params) *cobra.Command {
 	opts := options.NewLogOptions(p)
 
 	eg := `
-  # interactive mode: shows logs of the selected pipeline run
+Interactive mode: shows logs of the selected pipeline run:
```
Comment (danielhelfand) [inline pkg/cmd/pipeline/logs.go]:
```suggestion
Interactive mode: shows logs of the selected pipelinerun:
```

File: pkg/cmd/pipelineresource/list.go [hunk 2]
```diff
@@ -44,9 +44,9 @@ func listCommand(p cli.Params) *cobra.Command {
 
 	opts := &ListOptions{Type: ""}
 	f := cliopts.NewPrintFlags("list")
-	eg := `
-# List all PipelineResources in a namespaces 'foo'
-tkn pre list -n foo
+	eg := `List all PipelineResources in a namespaces 'foo':
```
Comment (danielhelfand) [inline pkg/cmd/pipelineresource/list.go]:
```suggestion
	eg := `List all PipelineResources in a namespace 'foo':
```

File: pkg/cmd/pipelinerun/cancel.go [hunk 3]
```diff
@@ -32,10 +32,10 @@ const (
 )
 
 func cancelCommand(p cli.Params) *cobra.Command {
-	eg := `
-  # cancel the PipelineRun named "foo" from the namespace "bar"
+	eg := `Cancel the PipelineRun named 'foo; from the namespace 'bar':
```
Comment (danielhelfand) [inline pkg/cmd/pipelinerun/cancel.go]:
```suggestion
	eg := `Cancel the PipelineRun named 'foo' from the namespace 'bar':
```

File: pkg/cmd/pipelinerun/list.go [hunk 4]
```diff
@@ -44,12 +44,13 @@ func listCommand(p cli.Params) *cobra.Command {
 
 	opts := &ListOptions{Limit: 0}
 	f := cliopts.NewPrintFlags("list")
-	eg := `
-# List all PipelineRuns of Pipeline name 'foo'
-tkn pipelinerun list foo -n bar
+	eg := `List all PipelineRuns of Pipeline name 'foo':
```
Comment (danielhelfand) [inline pkg/cmd/pipelinerun/list.go]:
```suggestion
	eg := `List all PipelineRuns of Pipeline 'foo':
```

File: pkg/cmd/pipelinerun/list.go [hunk 5]
```diff
@@ -44,12 +44,13 @@ func listCommand(p cli.Params) *cobra.Command {
...
 
-# List all pipelineruns in a namespaces 'foo'
-tkn pr list -n foo
+    tkn pipelinerun list foo -n bar
+
+List all pipelineruns in a namespaces 'foo':
```
Comment (danielhelfand) [inline pkg/cmd/pipelinerun/list.go]:
```suggestion
List all pipelineruns in a namespace 'foo':
```

File: pkg/cmd/pipelinerun/logs.go [hunk 6]
```diff
@@ -29,15 +29,16 @@ import (
 
 func logCommand(p cli.Params) *cobra.Command {
 	opts := &options.LogOptions{Params: p}
-	eg := `
-  # show the logs of PipelineRun named "foo" from the namesspace "bar"
+	eg := `Show the logs of PipelineRun named 'foo' from the namespace 'bar':
```
Comment (danielhelfand) [inline pkg/cmd/pipelinerun/logs.go]:
```suggestion
	eg := `Show the logs of PipelineRun named 'foo' from namespace 'bar':
```

File: pkg/cmd/pipelinerun/logs.go [hunk 7]
```diff
@@ -29,15 +29,16 @@ import (
...
+
     tkn pipelinerun logs foo -n bar
 
-  # show the logs of PipelineRun named "microservice-1" for task "build" only, from the namespace "bar"
+Show the logs of PipelineRun named 'microservice-1' for task 'build' only, from the namespace 'bar':
```
Comment (danielhelfand) [inline pkg/cmd/pipelinerun/logs.go]:
```suggestion
Show the logs of PipelineRun named 'microservice-1' for task 'build' only from the namespace 'bar':
```

File: pkg/cmd/pipelinerun/logs.go [hunk 8]
```diff
@@ -29,15 +29,16 @@ import (
...
+
     tkn pr logs microservice-1 -t build -n bar
 
-  # show the logs of PipelineRun named "microservice-1" for all tasks and steps (including init steps),
-    from the namespace "foo"
+Show the logs of PipelineRun named 'microservice-1' for all tasks and steps (including init steps), from the namespace 'foo':
```
Comment (danielhelfand) [inline pkg/cmd/pipelinerun/logs.go]:
```suggestion
Show the logs of PipelineRun named 'microservice-1' for all tasks and steps (including init steps) from namespace 'foo':
```

File: pkg/cmd/task/describe.go [hunk 9]
```diff
@@ -106,11 +106,13 @@ NAME	STARTED	DURATION	STATUS
 
 func describeCommand(p cli.Params) *cobra.Command {
 	f := cliopts.NewPrintFlags("describe")
-	eg := `
-# Describe a task of name 'foo' in namespace 'bar'
-tkn task describe foo -n bar
+	eg := `Describe a task of name 'foo' in namespace 'bar':
```
Comment (danielhelfand) [inline pkg/cmd/task/describe.go]:
```suggestion
	eg := `Describe a Task of name 'foo' in namespace 'bar':
```

File: pkg/cmd/task/logs.go [hunk 10]
```diff
@@ -46,20 +46,22 @@ func nameArg(args []string, p cli.Params) error {
 func logCommand(p cli.Params) *cobra.Command {
 	opts := options.NewLogOptions(p)
 
-	eg := `
-  # interactive mode: shows logs of the selected taskrun
+	eg := `Interactive mode: shows logs of the selected taskrun:
```
Comment (danielhelfand) [inline pkg/cmd/task/logs.go]:
```suggestion
	eg := `Interactive mode: shows logs of the selected TaskRun:
```

File: pkg/cmd/task/logs.go [hunk 11]
```diff
@@ -46,20 +46,22 @@ func nameArg(args []string, p cli.Params) error {
...
+
     tkn task logs -n namespace
 
-  # interactive mode: shows logs of the selected taskrun of the given task
+Interactive mode: shows logs of the selected taskrun of the given task:
```
Comment (danielhelfand) [inline pkg/cmd/task/logs.go]:
```suggestion
Interactive mode: shows logs of the selected TaskRun of the given task:
```

File: pkg/cmd/task/logs.go [hunk 12]
```diff
@@ -46,20 +46,22 @@ func nameArg(args []string, p cli.Params) error {
...
+
     tkn task logs task -n namespace
 
-  # show logs of given task for last taskrun
+Show logs of given task for last taskrun:
```
Comment (danielhelfand) [inline pkg/cmd/task/logs.go]:
```suggestion
Show logs of given Task for last TaskRun:
```

File: pkg/cmd/task/logs.go [hunk 13]
```diff
@@ -46,20 +46,22 @@ func nameArg(args []string, p cli.Params) error {
...
+
     tkn task logs task -n namespace --last
 
-  # show logs for given task and taskrun
-    tkn task logs task taskrun -n namespace
+Show logs for given task and taskrun:
```
Comment (danielhelfand) [inline pkg/cmd/task/logs.go]:
```suggestion
Show logs for given Task and TaskRun:
```

File: pkg/cmd/taskrun/list.go [hunk 14]
```diff
@@ -44,12 +44,13 @@ func listCommand(p cli.Params) *cobra.Command {
 
 	opts := &ListOptions{Limit: 0}
 	f := cliopts.NewPrintFlags("list")
-	eg := `
-# List all taskruns in namespace 'bar'
-tkn tr list -n bar
+	eg := `List all taskruns in namespace 'bar':
```
Comment (danielhelfand) [inline pkg/cmd/taskrun/list.go]:
```suggestion
	eg := `List all TaskRuns in namespace 'bar':
```

File: pkg/cmd/taskrun/list.go [hunk 15]
```diff
@@ -44,12 +44,13 @@ func listCommand(p cli.Params) *cobra.Command {
...
 
-# List all taskruns of task 'foo' in namespace 'bar'
-tkn taskrun list foo -n bar
+    tkn tr list -n bar
+
+List all taskruns of task 'foo' in namespace 'bar':
```
Comment (danielhelfand) [inline pkg/cmd/taskrun/list.go]:
```suggestion
List all TaskRuns of task 'foo' in namespace 'bar':
```

File: pkg/cmd/taskrun/logs.go [hunk 16]
```diff
@@ -33,12 +33,13 @@ const (
 
 func logCommand(p cli.Params) *cobra.Command {
 	opts := &options.LogOptions{Params: p}
-	eg := `
-# show the logs of TaskRun named "foo" from the namespace "bar"
-tkn taskrun logs foo -n bar
+	eg := `Show the logs of TaskRun named 'foo' from the namespace 'bar':
```
Comment (danielhelfand) [inline pkg/cmd/taskrun/logs.go]:
```suggestion
	eg := `Show the logs of TaskRun named 'foo' from namespace 'bar':
```

File: pkg/cmd/taskrun/logs.go [hunk 17]
```diff
@@ -33,12 +33,13 @@ const (
...
 
-# show the live logs of TaskRun named "foo" from the namespace "bar"
-tkn taskrun logs -f foo -n bar
+    tkn taskrun logs foo -n bar
+
+Show the live logs of TaskRun named 'foo' from the namespace 'bar':
```
Comment (danielhelfand) [inline pkg/cmd/taskrun/logs.go]:
```suggestion
Show the live logs of TaskRun named 'foo' from namespace 'bar':
```

**Comment by ImJasonH** [comment]
@danielhelfand I've resolved all the comments. It looks like there are still quite a few inconsistencies in examples and descriptions that should be audited and hammered out (e.g., `taskrun` vs `TaskRun` -- the latter being more standard in Tekton docs).

My main goal was to fix code samples and erroneous headings in Markdown, if someone wants to go through with an eye for consistency that could be a useful followup.

**Comment by ImJasonH** [comment]
/test pull-tekton-cli-unit-tests

(timed out after 10m)

**Pull request merged**